agent/src/main/java/com/microsoft/applicationinsights/agent/internal/config/XmlAgentConfigurationBuilder.java

```diff
@@ -110,9 +111,10 @@ public AgentConfiguration parseConfigurationFile(String baseFolder) {
                 try {
                     boolean debugMode = Boolean.valueOf(debugModeAsString);
                     agentConfiguration.setDebugMode(debugMode);
-                    InternalAgentLogger.INSTANCE.logAlways(InternalAgentLogger.LoggingLevel.ERROR, "Instrumentation debug mode set to '%s'", debugMode);
+                    InternalAgentLogger.INSTANCE.warn("Instrumentation debug mode set to '%s'", debugMode);
```

Comment:
IMO, this is an info message.

Reply:
Sure should be info. I dropped one level down (it was ERROR before). I will drop it down to INFO. Thanks for pointing out.